src/components/general/TermsOfUseDialog.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <v-dialog v-model="showTermsDialog" persistent max-width="900">
+    <v-card style="cursor: default">
+      <v-card-title class="text-h5">Terms of Use</v-card-title>
+      <HtmlDisplay :url="url" />
+      <v-card-actions class="pt-0">
+        <div class="d-flex flex-column w-100 mx-2">
+          <v-checkbox
+            v-model="isInAgreement"
+            label="I agree to the terms and conditions"
+            hide-details
+            density="comfortable"
+          />
+          <div class="d-flex mb-2">
+            <v-spacer />
+            <v-btn
+              @click="onAgreeClick"
+              variant="flat"
+              color="primary"
+              class="text-capitalize"
+              :disabled="!isInAgreement"
+            >
+              I Agree
+            </v-btn>
+          </div>
+        </div>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts" setup>
+import { useConfigStore } from '@/stores/config'
+import { computed, ref } from 'vue'
+import { useStorage } from '@vueuse/core'
+import HtmlDisplay from '@/components/general/HtmlDisplay.vue'
+
+const isInAgreement = ref(false)
+const showTermsDialog = useStorage(
```

Comment:
This uses session storage, no pinia

Reply:
Yes that is what I meant, do want to persist state with a mechanism other than Pinia? It may be more clear if all persisting is done via Pinia, rather than directly in storage.

Reply:
Ah like that, for now this is also how Werner did it for the splashscreen. I think since the stores are stored in localStorage and these in sessionStorage it is fine. But we could make a session stored store for these, since we now have 2.

Reply:
Discussed it with Tom, and am actually putting it in LocalStorage

Reply:
Alright, makes sense to keep a similar approach for both splash-screeny things.